src/form_builder/models.py

```diff
@@ -51,10 +51,12 @@ class Form(models.Model):
                              max_length=255,
                              help_text=_("Give your form a name."))
     slug = models.SlugField(_("Slug"),
```

Comment:
Yeah, you can confirm this by looking at how that string is used to change the label for the `confirmation_text` and `date_created` fields below.